results/pkg/watcher/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -119,6 +119,7 @@ func (tt *PipelineRunTest) testUpdatePipelineRun(t *testing.T) {
 		Executions: []*pb.Execution{{
 			Execution: &pb.Execution_PipelineRun{p},
 		}},
+		Etag: res.Etag,
```

Comment:
We should avoid tautological tests - this would pass regardless of the etag value. We should either generate this from the wanted result, or loosen the etag restriction to make it not required for requests.